packages/app-desktop/gui/NoteSearchBar.jsx

```diff
@@ -170,6 +173,7 @@ class NoteSearchBarComponent extends React.Component {
 					{allowScrolling ? previousButton : null}
 					{allowScrolling ? matchesFoundString : null}
 					{!allowScrolling ? viewerWarning : null}
+					{isSearchResultFound === false && <span>{ _('Phrase Not Found') }</span>}
```

Comment:
Take a look at the previous line, you should use the same structure for this line.
```
{!isSearchResultFound ? notFoundWarning : null}
```
You will have to define `notFoundWarning` above.

In general when contributing to a project, you will need to read through the codebase to understand how similar situations are formatted. While your solution works, it makes the code harder to read, which is going to prevent your change from being merged.

Reply:
Sorry, I'll fix it right away. 

Will keep in mind from next time.